backend/coins/btc/account.go

```diff
@@ -129,6 +131,7 @@ func NewAccount(
 	config *accounts.AccountConfig,
 	coin *Coin,
 	forceGapLimits *types.GapLimits,
+	getAddress func(blockchain.ScriptHashHex) *addresses.AccountAddress,
```

Comment:
please document what exactly this function should do, it's unclear if it returns addresses from any account, or from any account from the same keystore, or from *other* account of the same keystore, etc.

Reply:
I tried to give it a better name and add a comment, LMK how it looks like now